Models/PositionModels/action_mask_model.py

```diff
@@ -199,7 +199,7 @@ def _forward(self, input_dict, action=None):
         print(self.action_space)
```

Comment:
remove these print statements.

Reply:
oops!

Reply:
at first I thought "oops!", but that wasn't me. I can still remove them, but there are 5 print statements in total in this script. Do you want me to remove all of them?